t/16-utils.t

```diff
@@ -502,4 +503,17 @@ subtest 'human readable size' => sub {
     is(human_readable_size(245760), '240 KiB', 'less than a MB');
 };
 
+subtest 'create downloads list' => sub {
+    my $job_settings = {};
+    is_deeply create_downloads_list($job_settings), {}, 'empty for no settings provided';
+    $job_settings->{TEST_URL} = 'test';
+    $job_settings->{TEST} = 'test';
+    is_deeply create_downloads_list($job_settings), {}, '';
```

Comment:
This check should have a description. (The last parameter should not be empty.)

The same counts for the next check.

Reply:
I've added some labels, but they don't seem apt/concise. Do you have better suggestions?